kadi/events/models.py

```diff
@@ -1114,6 +1117,112 @@ class Eclipse(TlmEvent):
     fetch_event_msids = ['aoeclips', 'eb1k5', 'eb2k5', 'eb3k5']
 
 
+class StarAcq(TlmEvent):
+    """
+    ACA star acquisition
+
+    **Event definition**: interval where ``AOACASEQ == 'AQXN'``
+
+    **Fields**
+
+    ======== ========== ================================
+     Field      Type              Description
+    ======== ========== ================================
+      start   Char(21)   Start time (YYYY:DDD:HH:MM:SS)
+       stop   Char(21)    Stop time (YYYY:DDD:HH:MM:SS)
+     tstart      Float            Start time (CXC secs)
+      tstop      Float             Stop time (CXC secs)
+        dur      Float                  Duration (secs)
+    ======== ========== ================================
+    """
+    event_msids = ['aoacaseq']
+    event_val = 'AQXN'
+    event_min_dur = 20
+    lookback = 3
+
+    n_id = models.IntegerField(help_text='Number of identified acquisition stars', null=True)
+    id_status = models.CharField(max_length=8, null=True,
+                                 help_text='Identification status of catalog entries (T or F)')
+    one_shot_roll = models.FloatField(help_text='One shot update in roll (arcsec)', null=True)
+    one_shot_pitch = models.FloatField(help_text='One shot update in pitch (arcsec)', null=True)
+    one_shot_yaw = models.FloatField(help_text='One shot update in yaw (arcsec)', null=True)
+
+    @classmethod
+    def get_alerts(cls, event, event_msidset):
+        """
+        Check for large one-shot in pitch or yaw, or a small number of identified acq
+        stars.
+        """
+        def get_obsid():
+            dat = fetch.Msid('cobsrqid', event['tstop'] - 10, event['tstop'] + 300)
+            return int(dat.vals[0])
+
+        alerts = []
+
+        if abs(event['one_shot_pitch']) > 60 or abs(event['one_shot_yaw']) > 60:
```

Comment:
Is 60 a little low?  I thought the warning on 50220 wasn't that helpful with estimated maneuver error of ~53 arcsec.

```
One shot at 2017:081:20:16:31.008
Roll  : 2.11
Pitch : 63.38
Yaw   : 7.40
```